libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -43,6 +44,11 @@ class LnurlPayInfoJob(
         try {
             request = Json.decodeFromString<LnurlInfoRequest>(LnurlInfoRequest.serializer(), payload)
             val nodeState = breezSDK.nodeInfo()
+            val maxSendableMsats: ULong =
+                maxOf(
+                    ExtensionSettings.instance.autoChannelSetupFeeLimitMsats,
+                    nodeState.inboundLiquidityMsats
```

Comment:
I think this confuses the user acceptable fee limit with the max amount that can be received. 

If a fee limit is set, the max sendable can be nodeState.maxReceivableMsat (minus channelsBalanceMsat?). Then before creating the invoice in the next step, if the amount is over the channelsBalanceMsat check the openChannelFee result to see if the feeMsat is within the user fee limit.